app/services/solidus_bolt/payments/capture_service.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module SolidusBolt
+  module Payments
+    class CaptureService < SolidusBolt::BaseService
```

Comment:
CaptureService doesn't capture the payment. This would make a lot of confusion. Can you rename it?

Reply:
`CompleteOrPartialCaptureService` is a bit of a mouthful tho 🤔